tests/test-resource-groups/variables.tf

```diff
@@ -0,0 +1,28 @@
+variable "resource_query" {
```

Comment:
We can use locals in the test main.tf and just delete this variables.tf file

Reply:
resolved